src/app/map/map.component.ts

```diff
@@ -625,6 +625,7 @@ export class MapComponent implements OnInit {
     } 
     this.count ++;
     this.checkCount(this.count, 6);
+    this._mapService.map.fitBounds(this.firePerimeterLayer.getBounds(), {padding: [75,75]});
```

Comment:
There seem to be console errors thrown at this line, maybe we can check if there is a `firePerimeterLayer` before fitting bounds. 
![image](https://user-images.githubusercontent.com/59063181/208715118-d16ae058-a96b-4b0f-be77-a6a415c4643b.png)